pvt/benchmarks/src/PoolBenchmark.behavior.ts

```diff
@@ -212,6 +274,9 @@ export class Benchmark {
       sharedBeforeEach('approve router to spend BPT', async () => {
         const bpt: IERC20 = await TypesConverter.toIERC20(this.pool);
         await bpt.connect(alice).approve(router, MAX_UINT256);
+        await bpt.connect(alice).approve(batchRouter, MAX_UINT256);
+        await bpt.connect(alice).approve(permit2, MAX_UINT256);
+        await permit2.connect(alice).approve(await bpt.getAddress(), batchRouter, MAX_UINT160, MAX_UINT48);
```

Comment:
```suggestion
        await bpt.connect(alice).approve(permit2, MAX_UINT256);
        await permit2.connect(alice).approve(bpt, batchRouter, MAX_UINT160, MAX_UINT48);
```

`BatchRouter` always uses `permit2`, so you shouldn't need to give BPT allowance directly.